src/ddmd/dclass.d

```diff
@@ -242,6 +240,10 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
     /// to prevent recursive attempts
     private bool inuse;
 
+    /// true if this class has an identifier, but was originally declared anonymous
```

Comment:
Do we really need to mention the second part, after the comma?

Reply:
IMO, yes.